src/simulate.jl

```diff
@@ -271,10 +270,7 @@ function simulate(JADEmodel::JADEModel, parameters::JADESimulation)
                     if (t + d.rundata.start_wk - 2) % WEEKSPERYEAR == WEEKSPERYEAR - 1
                         i += 1
                     end
-                    push!(
-                        sample_paths[end],
-                        ((t + d.rundata.start_wk - 2) % WEEKSPERYEAR + 1, s_inflows),
-                    )
+                    push!(sample_paths[end], (t, s_inflows))
```

Comment:
The node in the sample path needs to be the node in the SDDP graph. I think `t` here is `1:20`, where as the previous was the actual week, `vcat(45:52,1:13)`? Is it true that nodes in SDDP are always `1..T`?

Reply:
Something about this looks messed up, but perhaps there's something tricky going on that I've forgotten about. I'll dig into tomorrow; don't spend any time on this in the meantime.

Reply:
I found the commit that broke this on January 26 2022, from the private JADE repo.

I think that it needs to be:
`push!(sample_paths[end], ((t - 1) % WEEKSPERYEAR + 1, s_inflows))`,

in order to be able to wrap around for infinite-horizon models (i.e. if we wish to simulate an infinite horizon model from stage 26, for 52 weeks, then the stages should be 26,...,52,1,...,25. However, I need to do some tests to ensure that errors are being appropriate thrown if the model isn't set up right.

Reply:
How do I create a test that will trip this? I'm very confused as to how the JADE and SDDP week/indices line up.

Reply:
The `rundata.start_wk` parameter is the first week of the model, this will correspond to stage 1 of the SDDP model.

The `parameters.initial_stage` is the starting stage of the simulation (the initial states will be injected into this stage and the simulation will continue from this stage). This could have been made easier for the user and implemented as `initial_week`, but it will getting quite confusing...so this wasn't done. The `initial_stage` parameter will typically be left at 1.

I should be able to create a test for this; it probably can be done with a one/two period model, and the cost of an untrained model will change as the parameters vary.